worlds/cvcotm/aesthetics.py

```diff
@@ -0,0 +1,779 @@
+    return location_bytes
+
+
+def populate_enemy_drops(world: "CVCotMWorld") -> Dict[int, bytes]:
+    """Randomizes the enemy-dropped items throughout the game within each other. There are three tiers of item drops:
+    Low, Mid, and High. Each enemy has two item slots that can both drop its own item; a Common slot and a Rare one.
+
+    On Normal item randomization, easy enemies (below 61 HP) will only have Low-tier drops in both of their stats,
+    bosses and candle enemies will be guaranteed to have High drops in one or both of their slots respectively (bosses
+    are made to only drop one slot 100% of the time), and everything else can have a Low or Mid-tier item in its Common
+    drop slot and a Low, Mid, OR High-tier item in its Rare drop slot.
+
+    If Item Drop Randomization is set to Tiered, the HP threshold for enemies being considered "easily" will raise to
+    below 144, enemies in the 144-369 HP range (inclusive) will have a Low-tier item in its Common slot and a Mid-tier
+    item in its rare slot, and enemies with more than 369 HP will have a Mid-tier in its Common slot and a High-tier in
+    its Rare slot. Candles and bosses still have Rares in all their slots, but now the guaranteed drops that land on
+    bosses will be exclusive to them; no other enemy in the game will have their item.
+
+    This and select_drop are the most directly adapted code from upstream CotMR in this package by far. Credit where
+    it's due to Spooky for writing the original, and Malaert64 for further refinements and updating what used to be
+    Random Item Hardmode to instead be Tiered Item Mode. The original C code this was adapted from can be found here:
+    https://github.com/calm-palm/cotm-randomizer/blob/master/Program/randomizer.c#L1028"""
+
+    placed_low_items = [0] * len(LOW_ITEMS)
+    placed_mid_items = [0] * len(MID_ITEMS)
+    placed_high_items = [0] * len(HIGH_ITEMS)
+
+    placed_common_items = [0] * len(COMMON_ITEMS)
+    placed_rare_items = [0] * len(RARE_ITEMS)
+
+    regular_drops = [0] * NUMBER_ENEMIES
+    regular_drop_chances = [0] * NUMBER_ENEMIES
+    rare_drops = [0] * NUMBER_ENEMIES
+    rare_drop_chances = [0] * NUMBER_ENEMIES
+
+    # Set boss items first to prevent boss drop duplicates.
+    # If Tiered mode is enabled, make these items exclusive to these enemies by adding an arbitrary integer larger
+    # than could be reached normally (e.g.the total number of enemies) and use the placed high items array instead of
+    # the placed rare items one.
+    if world.options.item_drop_randomization == ItemDropRandomization.option_tiered:
+        for boss_id in BOSS_IDS:
+            regular_drops[boss_id] = select_drop(world, HIGH_ITEMS, placed_high_items, True)
+    else:
+        for boss_id in BOSS_IDS:
+            regular_drops[boss_id] = select_drop(world, RARE_ITEMS, placed_rare_items, start_index=len(COMMON_ITEMS))
+
+    # Setting drop logic for all enemies.
+    for i in range(NUMBER_ENEMIES):
```

Comment:
That enemy data list didn't exist in the initial version of the enemy randomizer code in both my version and upstream CotMR's version, and I kinda didn't think to remove that constant after the fact, I guess.

Reply:
Fair. I mean it's not a problem. If code parity is a goal so that you can port future features easier, then this has more justification to stick around.

Reply:
A change like this is pretty in-line with the slight adjustments I made to clean it up in places where I saw fit, so I'll take it.